### ISSUE TITLE
New version generates error in dashboard for installations with modified core folder

### ISSUE THREAD

**Issue by SnowCreative** [BobRay/UpgradeMODX#70]
After upgrading to UpgradeMODX version 2.1.4, the dashboard widget no longer works, and logging into the manager gets stalled. Here's the error:

Fatal error: require_once(): Failed opening required '.../core/components/upgrademodx/vendor/autoload.php' (include_path='.:/opt/alt/php72/usr/share/pear') in .../modxcore/components/upgrademodx/model/upgrademodx/upgrademodx.class.php on line 182

Did you hard code the core path? My core is renamed and outside the root, hence the failure to load a file.

**Comment by BobRay** [comment]
Not exactly, but something similar. I used MODX_BASE_PATH instead of MODX_CORE_PATH. See #69 for the fix.
